src/BitStringAddresses/fermifs.jl

```diff
@@ -131,6 +131,22 @@ Base.:(==)(a::FermiFS, b::FermiFS) = a.bs == b.bs
 num_occupied_modes(::FermiFS{N}) where {N} = N
 occupied_modes(a::FermiFS{N,<:Any,S}) where {N,S} = FermiOccupiedModes{N,S}(a.bs)
 
+num_unoccupied_modes(::FermiFS{N,M}) where {N,M} = M - N
+unoccupied_modes(a::FermiFS{N,<:Any,S}) where {N,S} = FermiUnoccupiedModes{N,S}(~a.bs)
+
+function UnoccupiedModeMap(addr::FermiFS{N,M}) where {N,M}
```

Comment:
In Julia we use CamelCase typically only for types. Functions can be CamelCase when they return a type of the same name (i.e. are type constructors). Here you have a CamelCase'd function that returns a type with a different name, and we should avoid this.

As a suggested resolution, just use snake_case for the name of the function:
```suggestion
function unoccupied_mode_map(addr::FermiFS{N,M}) where {N,M}
```
It is still a bit weird that your function returns a type `OccupiedModeMap`, which however stores a map of the unoccupied modes. Maybe we should rename  `OccupiedModeMap` into `ModeMap`, as it can hold any type of modes (occupied or not)?